picasso/src/test/java/com/squareup/picasso/BitmapHunterTest.java

```diff
@@ -1162,4 +1164,10 @@
       return new Result(bitmap, MEMORY);
     }
   }
+
+  private static final Call.Factory UNUSED_CALL_FACTORY = new Call.Factory() {
```

Comment:
Delete and use the one in `TestUtils`

Reply:
TestUtils only has a fake `OkHttp3Downloader`, but we could pull out the `Call.Factory` and use that here.  👍

Reply:
This PR does that. It defines it in both places.

Reply:
Ah, missed that.  👍